### PR TITLE
feat: add back `getUrl` & `getDataUrl`

### PR DESCRIPTION
**Pull request by stipsan**

### CODE REVIEW

File: src/SanityClient.ts [hunk 1]
```diff
@@ -613,6 +613,26 @@ export class ObservableSanityClient {
   request<R = FIXME>(options: RawRequestOptions): Observable<R> {
     return dataMethods._request(this, this.#httpRequest, options)
   }
+
+  /**
+   * Get a Sanity API URL for the URI provided
+   *
+   * @param uri URI/path to build URL for
+   * @param canUseCdn Whether or not to allow using the API CDN for this route
```
Comment (rexxars) [inline src/SanityClient.ts]:
```suggestion
   * @param uri - URI/path to build URL for
   * @param canUseCdn - Whether or not to allow using the API CDN for this route
```

File: src/SanityClient.ts [hunk 2]
```diff
@@ -613,6 +613,26 @@ export class ObservableSanityClient {
...
+   */
+  getUrl(uri: string, canUseCdn?: boolean): string {
+    return dataMethods._getUrl(this, uri, canUseCdn)
+  }
+
+  /**
+   * Get a Sanity API URL for the data operation and path provided
+   *
+   * @param operation Data operation
+   * @param path Path to append
```
Comment (rexxars) [inline src/SanityClient.ts]:
```suggestion
   * @param operation - Data operation (eg `query`, `mutate`, `listen` or similar)
   * @param path - Path to append after the operation
```

File: src/SanityClient.ts [hunk 3]
```diff
@@ -1210,4 +1230,24 @@ export class SanityClient {
   dataRequest(endpoint: string, body: unknown, options?: BaseMutationOptions): Promise<FIXME> {
     return lastValueFrom(dataMethods._dataRequest(this, this.#httpRequest, endpoint, body, options))
   }
+
+  /**
+   * Get a Sanity API URL for the URI provided
+   *
+   * @param uri URI/path to build URL for
+   * @param canUseCdn Whether or not to allow using the API CDN for this route
```
Comment (rexxars) [inline src/SanityClient.ts]:
```suggestion
   * @param uri - URI/path to build URL for
   * @param canUseCdn - Whether or not to allow using the API CDN for this route
```

File: src/SanityClient.ts [hunk 4]
```diff
@@ -1210,4 +1230,24 @@ export class SanityClient {
...
+   */
+  getUrl(uri: string, canUseCdn?: boolean): string {
+    return dataMethods._getUrl(this, uri, canUseCdn)
+  }
+
+  /**
+   * Get a Sanity API URL for the data operation and path provided
+   *
+   * @param operation Data operation
+   * @param path Path to append
```
Comment (rexxars) [inline src/SanityClient.ts]:
```suggestion
   * @param operation - Data operation (eg `query`, `mutate`, `listen` or similar)
   * @param path - Path to append after the operation
```

**Review by rexxars — CHANGES_REQUESTED** [review]
Just a few TSDoc compatibility changes, otherwise this looks great! Thanks!

**Review by bjoerge — APPROVED** [review]
LGTM! ✨

**Pull request merged**